src/components/map/buttons/NavigationButtons.vue

```diff
@@ -28,13 +28,28 @@ const panelStore = usePanelsStore()
 const mapStore = useMapStore()
 const simulationStore = useSimulationStore()
 
+function IsPanelVisibleOnStep() {
+  if (
+    (simulationStore.currentStep == 2 && simulationStore.currentSubStep == 2) ||
+    (simulationStore.currentStep == 3 && simulationStore.currentSubStep == 1) ||
+    (simulationStore.currentStep == 3 && simulationStore.currentSubStep == 2)
+  )
+    return true
+}
+
 async function zoom(out = false, zoomFactor = 2): Promise<void> {
   const activeMap = rennesApp.maps.activeMap
   const viewpoint = await activeMap?.getViewpoint()
   const maxZoom = rennesApp.get3DMap().getScene()
     .screenSpaceCameraController.maximumZoomDistance
   if (activeMap && viewpoint) {
     let distance = viewpoint.distance / zoomFactor
+    if (
+      [viewList['step-sunshine']].includes(viewStore.currentView!) &&
```

Comment:
You can but this condition [viewList['step-sunshine']].includes(viewStore.currentView!) in the function IsPanelVisibleOnStep